### PR TITLE
Jeremy sir this is my Lab 2 part 1, please accept my pull request, give any feedback!

### PR DESCRIPTION
**Pull request by pparasuram**
Jeremy sir this is my Lab 2 part 1, please accept my pull request, give any feedback!

Prakash Parasuram

### CODE REVIEW

File: pparasuram-interview-lab/Interview.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Prakash Parasuram: GitHub Lab Part Two:
```
Comment (jschmersal-cscc) [inline pparasuram-interview-lab/Interview.md:1]: It looks like you've accidentally added your interview to this PR.  Please remove this file from the PR and update it.

File: README.md [hunk 2]
```diff
@@ -31,14 +31,28 @@ Q2: What is the answer?
...
 
-Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+* _Prakash_: I started off as a programmmer and took a detour into networking, I am a Wireless Network Engineer now, but so much excitement has happened in the programming world, I wish to get back in the know, and possibly go back to programming.  
 
-Q3: Name your favorite member of [The Beatles](https://en.wikipedia.org/wiki/The_Beatles), and your favorite Beatles song.  You do know at least one Beatles song, right?  If you are a Beatles fan, feel free to give me your top 5 or so.
+Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?  
 
-Q4: Do you have any pets? Species/Name/Details please. [Murray](images/Murray.jpeg?raw) wants to know... (he's not as smart as that picture makes him seem). 
+* _Prakash_: I used to be a merchant marine ship engineer and rose to the rank of a 3rd engineer, so I have traveled a bit, except that most times the stay is short, Greece seems to be my favorite destination, based on what I have read so far.
```
Comment (jschmersal-cscc) [inline README.md:40]: Did you stay anywhere long enough to know that you liked it?

**Review by jschmersal-cscc — CHANGES_REQUESTED** [review]
I have a couple changes I'd like you to do and then you'll be good to go.

**Comment by pparasuram** [comment]
Please see if final changes are what you wanted.

**Comment by jschmersal-cscc** [comment]
Looks good!  Go ahead and submit in blackboard!

**Pull request closed without merging**